cpp/src/arrow/status.h

```diff
@@ -176,7 +176,10 @@ class ARROW_EXPORT Status {
   void CopyFrom(const State* s);
 };
 
-std::ostream& operator<<(std::ostream& os, const Status& x);
```

Comment:
See https://github.com/conda-forge/pyarrow-feedstock/pull/33#issuecomment-316075145, it seems like this is not the problematic function.